src/componets/Weather/weather.tsx

```diff
@@ -1,46 +1,44 @@
 import React from 'react';
-import GetRequestAsyncAwait from '../request/requestko'
 import Box from "./Box/Box";
 import Grid from "./Grid/Grid";
 import GridRuler from "./GridRuler/GridRuler";
+import GetWeatherData,{WeatherDataGet}from '../API/weather_query';
+import Row from '../request/test';
+
+
 
 
 function Weather() {
+  // console.log(Day())
+  let data = WeatherDataGet()
+  // let current:string = data.state
+
   return (
     <div
       className="Weather"
-      style={{ margin: "16px", position: "relative", height: "100vh" }}
+      style={{ 
```

Comment:
Don't do styling inside Javascript.. This would hurt the performance..

Reply:
done na po nilipat ko po sa scss tama po ba